docs/tutorials/enterprise_beginner/ez_enterprise_beginner_tutorial_-_its_a_dogs_world.md

```diff
@@ -12,27 +12,27 @@ This tutorial is intended for a beginner with basic knowledge of the workings of
 
 After going through this tutorial, you will:
 
-- Have a working knowledge of Landing Page functionality and architecture
+- Have a working knowledge of a Landing Page functionality and architecture
```

Comment:
I think `the Landing Page` if we want an article at all, as there is only one "functionality and architecture" for LP